### DISCUSSION TITLE
Is there any documentation on the commandline options?

### DISCUSSION THREAD

**Discussion by chengguizi** [eclipse-ecal/ecal#840] [Q&A] [answered]
In eCAL::Initialize(), we could pass in commandline argc argv arguments, but I am not sure what are the useful arguments we could possibly pass in?

Reading 
- https://github.com/eclipse-ecal/ecal/blob/master/ecal/core/src/ecal.cpp#L89-L91

seems to give some of them. Are we expected to use them, to change how eCAL node is behaving?

**Comment by rex-schilasky** [comment, marked as answer]
Ye, this is not well documented. At least with `--help` eCAL will give you the available command line options.

![image](https://user-images.githubusercontent.com/49162693/196631173-17576f82-c656-462c-a4fd-e5cd28e76e75.png)

so you can specify another ecal configuration file or overwrite single options from the configuration with a `section/key:value` syntax.
